api/v2/openapi.yaml

```diff
@@ -505,6 +517,29 @@ definitions:
       - state
       - silencedBy
       - inhibitedBy
+  alertGroupAlertStatus:
```

Comment:
This was duplicated to fix https://github.com/prometheus/alertmanager/pull/3797#discussion_r1550045174.